react/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "vtex.styleguide",
+    "dependencies": {
+        "react-responsive-modal": "^2.0.1",
+        "vtex-tachyons": "^2.3.0",
+        "@vtex/styleguide": "~2.3.0",
+        "apollo-client": "~2.2.3",
+        "graphql": "~0.13.2",
```

Comment:
In an APP, these dependencies are not necessary...
`
vtex-tachyons, graphql, @vtex/styleguide, apollo-client, graphql, graphql-tag, react-apollo, react-intl, react, react-dom
`